hack/lib/protoc.sh

```diff
@@ -27,7 +27,7 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 # $1: Full path to the directory where the api.proto file is
 function kube::protoc::generate_proto() {
   kube::golang::setup_env
-  go install k8s.io/kubernetes/vendor/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
+  GO111MODULE=on GOPROXY=off go install k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
```

Comment:
do we use  @latest always?

Reply:
I see  https://go.dev/ref/mod

> In GOPATH mode, the go command ignores modules; it looks in [vendor directories](https://go.dev/ref/mod#glos-vendor-directory) and in GOPATH to find dependencies.

Reply:
ok, scratch my comment

> If the arguments don’t have version suffixes, go install may run in module-aware mode or GOPATH mode, depending on the GO111MODULE environment variable and the presence of a go.mod file. See [Module-aware commands](https://go.dev/ref/mod#mod-commands) for details. If module-aware mode is enabled, go install runs in the context of the main module, which may be different from the module containing the package being installed.

Reply:
yeah, we want to build the versions at HEAD / in `vendor` ... we explicitly do *not* want to reach out to the network to resolve code and install these commands